drafts/GoL-reules/Game of Links.md

```diff
@@ -0,0 +1,139 @@
+# The Game of Links. Goals, rules and provision.
+
+## Intro
```

Comment:
I think we should clearly translate that the purpose of the game: to prepare and test infrastructure for main network